app.js

```diff
@@ -26,6 +27,8 @@ app.post("/api/articles/:article_id/comments", postComment);
 
 app.patch("/api/articles/:article_id", patchArticle);
 
+app.delete("/api/comments/:comment_id", deletedComment);
```

Comment:
It's a small detail but the function name `deletedComment` suggests the action of deleting a comment has happened, the function name should represent what it's _going_ to do, so changing this to `deleteCommentById` would be more descriptive.